src/test/java/hw2/base/BaseTest.java

```diff
@@ -0,0 +1,155 @@
+        driver = new ChromeDriver();
+        driver.manage().window().maximize();
+    }
+
+    @AfterClass
+    public void tearDown() {
+        driver.quit();
+    }
+
+    //ex1 and ex2 common steps
+    public void openURL() {
+        driver.get(BASE_URL);
+    }
+
+    public void assertTitle(String titleExpected) {
+        assertEquals(driver.getTitle(), titleExpected);
+    }
+
+    public void performLogin(LoginUser login) {
+        driver.findElement(By.className("profile-photo")).click();
+        driver.findElement(By.id("name")).sendKeys(login.getLogin());
+        driver.findElement(By.cssSelector("#password")).sendKeys(login.getPassword());
+        driver.findElement(By.id("login-button")).click();
+    }
+
+    public void assertUserName(LoginUser username) {
+        WebElement element = driver.findElement(By.xpath("//*[@class='profile-photo']//span"));
+        assertTrue(element.isDisplayed());
+        assertEquals(element.getText(), username.getUsername());
+    }
+
+    //ex1 steps
+    public void assertItemsOnTheHeaderSection(WebElement assertionData, String expectedHeaders) {
+        assertTrue(assertionData.isDisplayed());
+        assertEquals(assertionData.getText(), expectedHeaders);
+    }
+
+    public void checkImagesExistence(List<WebElement> images) {
+        Assert.assertEquals(images.size(), 4);
+        for (WebElement img : images) {
+            Assert.assertTrue(img.isDisplayed());
+        }
+    }
+
+    public void checkTextBelowIcons(List<WebElement> texts) {
+        assertEquals(texts.size(), 4);
+        for (WebElement text : texts) {
+            assertTrue(text.isDisplayed());
+        }
+    }
+
+    public void assertFrameButton() {
+        WebElement frame = driver.findElement(By.id("frame"));
+        boolean existedFrame = frame.isEnabled();
+        Assert.assertTrue(existedFrame);
+    }
+
+    public void switchToTheFrameAndCheck() {
+        WebElement frame = driver.findElement(By.id("frame"));
+        driver.switchTo().frame(frame);
+        WebElement iFrameButton = driver.findElement(By.id("button-frame"));
+        assertTrue(iFrameButton.isDisplayed());
+    }
+
+    public void switchToOriginalWindowBack() {
+        driver.switchTo().defaultContent();
+    }
+
+    public void assertItemsOnTheLeftSection(WebElement assertionData, String expectedHeaders) {
+        assertTrue(assertionData.isDisplayed());
+        assertEquals(assertionData.getText(), expectedHeaders);
+    }
+
+    //ex2 steps
+    public void openDifferentElementsPage() {
+        WebElement element = driver.findElement((By.xpath("//a/span[contains(text(),'Service')]")));
+        element.click();
+        driver.findElement((By.xpath("//a[contains(text(), 'Different elements')]")));
+        element.click();
+        driver.get("https://jdi-testing.github.io/jdi-light/different-elements.html");
```

Comment:
I will repeat myself. You must not use driver.get(...) anywhere except to open main page.
In this method you must navigate via selenium actions.
Let's imagine: you test navigation by click on header. You click and make driver.get(...). BUT it doesn't fit to "user actions", user will have no driver to navigate. Got it?



This method should be smth like:
1.  open 'Services' tab
2.  click 'Different Elements'

Not:
1.  open 'Services' tab
2.  click 'Different Elements'
3. driver.get()

Anyway, error in :
  element.click(); <------- this line not work properly.
![image](https://user-images.githubusercontent.com/44112289/102100941-15859280-3e3b-11eb-89b3-80a2088eee45.png)

you twice click by 'Services' because you not update value in variable `element`